src/groups/mqb/mqbstat/mqbstat_queuestats.h

```diff
@@ -110,8 +110,10 @@ class QueueStatsDomain {
             e_NB_CONSUMER,
             e_MESSAGES_CURRENT,
             e_MESSAGES_MAX,
+            e_MESSAGES_UTILIZATION,
```

Comment:
What do you think about ending with "_MAX" to make it clear it's peak utilization? Otherwise, lgtm.

Reply:
Agree, added suffix `_MAX` to the new metrics.